### ISSUE TITLE
Log In button should show and external link icon

### ISSUE THREAD

**Issue by Sebastian-ubs** [sillsdev/scripture-forge-platform-extensions#14]
According to the newly added guideline https://github.com/paranext/paranext-core/blob/main/lib/papi-dts/papi.d.ts#L3140
the "Log In" button should have an `ExternalLink` icon and some `aria-label`
